go-apps/meep-platform-ctrl/server/user_authentication.go

```diff
@@ -140,3 +140,18 @@ func uaLogoutUser(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
 	w.WriteHeader(http.StatusOK)
 }
+
+func uaRefreshUser(w http.ResponseWriter, r *http.Request) {
+	log.Info("----- KEEP-ALIVE -----")
```

Comment:
We should remove this log... it will be noisy.

Reply:
Removed.